tests/test_bulk_tx_basic/src/main.xc

```diff
@@ -13,7 +12,8 @@
 #define EP_COUNT_OUT   		(6)
 #define EP_COUNT_IN    		(6)
 
-/* Endpoint type tables */
+#include "shared.h"
+
 XUD_EpType epTypeTableOut[EP_COUNT_OUT] = {XUD_EPTYPE_CTL,
                                                 XUD_EPTYPE_BUL,
                                                  XUD_EPTYPE_BUL,
```

Comment:
It appears that this line has a different indentation to the ones surrounding it.  Is that intentional?

Reply:
Fixed